ast.go

```diff
@@ -1086,8 +1137,9 @@ func (*ColumnConstraintGenerated) iColumnConstraint()  {}
 
 // ColumnConstraintPrimaryKey represents a PRIMARY KEY column constraint for CREATE TABLE.
 type ColumnConstraintPrimaryKey struct {
-	Name  Identifier
-	Order string
+	Name          Identifier
+	Order         string
+	AutoIncrement bool
```

Comment:
Adds the `AutoIncrement` flag in the `ColumnConstraintPrimaryKey` node. That is the flag that will be forced in the grammar according to our spec rules.